vue-app/src/components/ProjectProfile.vue

```diff
@@ -262,6 +263,10 @@ export default class ProjectProfile extends Vue {
       }
     )
   }
+
+  private get blockExplorer(): string {
```

Comment:
For the sake of consistency let's go with `blockExplorerUrl` getter here where we construct the full URL in the getter.